### ISSUE TITLE
installing karabiner-elements removes alt

### ISSUE THREAD

**Issue by jmhodges** [tekezo/Karabiner-Elements#1304]
After installing and running karabiner-elements 11.6.0, my Alt key's on my Microsoft Natural Keyboard stop being mapped to Command and setting "left_command" to "left_command" doesn't fix it.

What's the fix here?

**Comment by wawe** [comment]
I encountered the same issue, also using a Microsoft Natural Ergonomic 4000 keyboard.
It started after I installed the High Sierra update to 10.13.3 and vanished after installing the 10.13.3 Supplemental Update.

I was running Karabiner-Elements 11.6.0 before and after the macOS updates, so at least my issue was not related to the 11.6.0 release.

Might just be a coincidence, since installing the update also rebooted the system etc. According to the release notes the supplemental update fixed issues with certain character sequences crashing apps, so it seems at least vaguely related to keyboard handling.

@jmhodges, did you also install any macOS updates recently?
